src/main/java/core/basesyntax/service/impl/FruitListImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
The import statement for 'FruitTransaction' seems to be incorrect if it's meant to be a model class. It should be placed in a 'model' package according to the checklist.